exporters/aerospike/aerospike-config.yml.sample

```diff
@@ -0,0 +1,26 @@
+integrations:
+    - name: nri-aerospike
+      config:
+        # API URL of the aerospike service
+        aerospike_db_host: localhost
```

Comment:
why having the "aerospike_db" prefix? would just:
"host", "port" work similarly to [ibmmq](https://github.com/newrelic/newrelic-prometheus-exporters-packages/blob/b8eaec72564df8c4c913f3a984e7e080a3de3699/exporters/ibmmq/ibmmq-config.yml.sample#L5) ?

Reply:
actually I noticed that mongodb and powerDNS have prefixes, therefore if you prefer them like that it is fine :)

Reply:
the value on "aerospike_db_host" will be assigned to "db_host" present in "ape.toml.tmpl" file.
Hence, I have named it like "aerospike_db_host"